src/test/java/com/google/javascript/clutz/DeclarationGeneratorTests.java

```diff
@@ -77,7 +77,8 @@ public static TestSuite suite() throws IOException {
       if (input.getName().contains("_output_base")) {
         subject.emitBase = true;
       }
-      if (Arrays.asList("partial", "multifilePartial").contains(input.getParentFile().getName())) {
+      if (Arrays.asList("partial", "multifilePartial", "crossModuleImports")
```

Comment:
It's kinda unpredictable from the outside that "crossModuleImports" implies "partial".  Maybe it should be "partialCrossModuleImports" or something?

Reply:
Sounds good!